contracts/modules/LoanOpenings/LoanOpenings.sol

```diff
@@ -23,15 +23,16 @@ contract LoanOpenings is State, LoanOpeningsEvents, VaultController, InterestHan
     {
         _setTarget(this.borrowOrTradeFromPool.selector, target);
         _setTarget(this.setDelegatedManager.selector, target);
-        _setTarget(this.getEstimatedMarginExposure.selector, target); // DEPRECATED, remove next deploy
-        _setTarget(this.getRequiredCollateral.selector, target);
-        _setTarget(this.getRequiredCollateralByParams.selector, target);
-        // _setTarget(this.getBorrowAmount.selector, target);
-        // _setTarget(this.getBorrowAmountByParams.selector, target);
         _setTarget(this.getDefaultLoanParams.selector, target);
         _setTarget(this.getPoolLoanParamId.selector, target);
         _setTarget(this.getLoanParamId.selector, target);
-        
+
+        // TODO remove next deploy ? shall I even remove them if they are read only?
```

Comment:
remove it to keep it clean on the used sigs. Its better to always maintain it and make sure it is up to date.